packages/docs-data/compile-docs-data.mjs

```diff
@@ -58,17 +69,17 @@ async function generateDocumentalistData() {
             new TypescriptPlugin({
                 excludeNames: [/.+State$/],
                 excludePaths: ["node_modules/", "-app/", "test-commons/", "-build-scripts/", "test/"],
-                // tsconfigPath: resolve(monorepoRootDir, "./config/tsconfig.node.json"),
                 verbose: true,
             }),
         )
         .use(".scss", new KssPlugin())
         .use("package.json", new NpmPlugin());
 
     const docs = await documentalist.documentGlobs(
-        "../*/src/**/*.{scss,md}",
-        "../*/src/index.{ts,tsx}",
-        "../*/package.json",
+        `../{${LIBRARY_AND_DOCS_PACKAGES.join(",")}}/src/**/*.md`,
+        `../{${LIBRARY_PACKAGES.join(",")}}/src/**/*.scss`,
+        `../{${LIBRARY_PACKAGES.join(",")}}/src/index.ts`,
+        `../{${LIBRARY_PACKAGES}}/package.json`,
```

Comment:
filtering to only the relevant packages saves a lot of time (about 70 seconds, or 40% on my machine)